evennia/contrib/worldloader/README.md

```diff
@@ -0,0 +1,126 @@
+# Intro
+It is a tool for Evennia(http://github.com/evennia/evennia) which can load game world from CSV files.
+
+
+## Installation
+You need to install Evennia and create your game first. Evennia's wiki is here: http://github.com/evennia/evennia/wiki.
+
+Copy the folder ```/example_tutorial_world/worlddata``` to your game folder. The directory should look like
```

Comment:
If a player want to create his own game world, he will edit these CSV files and even edit db models. So it's better to put these files in the game folder.